docs/01-get-started/your-first-route.mdx

```diff
@@ -6,12 +6,48 @@ sidebar_position: 3
 
 <p>{frontMatter.description}</p>
 
-:::info WIP
+# Routes
 
-Add instructions here:
+Starting v3.0.0, Front-Commerce uses Remix's
+[API Routes](https://remix.run/docs/en/main/guides/api-routes) to handle all the
+routing of you application. All routes are located under `app/routes` folder.
 
-- Create a route
-- Introduce loader
-- Redirect to Remix guides
+API Routes will use the default export of each Route file to generate the
+route's UI. Go ahead and create a new route `hello-world.tsx`:
 
-:::
+```tsx title="app/routes/hello-world.tsx"
+export default function HelloWorld() {
+  return (
+    <div>
+      <h1>Hello world!</h1>
+    </div>
+  );
+}
+```
+
+# Loaders
+
+In order to display dynamic content in your routes, you will need to fetch some
+data. This is done by using Remix's Routes API method `loader`.
+
+Here is a basic example of how a `loader` could be used:
+
+```tsx title="app/routes/hello-world.tsx"
+export const loader = () => {
+  // This method will be called prior to rendering to fetch the data for your component.
+  return { name: "Jack Doe" };
+};
+
+export default function HelloWorld() {
+  const { name } = useLoaderData();
+
+  return (
+    <div>
+      <h1>Hello {name}!</h1>
+    </div>
+  );
+}
+```
+
+You can learn more about this API in
+[Remix's Documentation](https://remix.run/docs/en/main/guides/api-routes).
```

Comment:
**issue:** there are much more thing to share here, and I think we must say that we won't cover everything because we're just using native Remix features here

**suggestion:** add a `External resources` section, with several other resources that we could enrich / reevaluate over time (e.g: the one sent by Paul, https://remix.run/docs/en/main/start/tutorial#the-contact-route-ui etc…).

There is also https://interactive-remix-routing-v2.netlify.app/ which is great!